plugins/modules/idrac_secure_boot.py

```diff
@@ -0,0 +1,399 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Dell OpenManage Ansible Modules
+# Version 9.6.0
+# Copyright (C) 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: idrac_secure_boot
+short_description: Import secure boot certificate.
+version_added: "9.6.0"
+description:
+  - This module allows to import the secure boot certificate.
+extends_documentation_fragment:
+  - dellemc.openmanage.idrac_x_auth_options
+options:
+  import_certificates:
+    type: bool
+    description:
+        - Import all the specified key certificates.
+        - When I(import_certificates) is C(true) either of I(platform_key) or I(KEK) or I(database) or I(disallow_database) is required.
```

Comment:
- When I(import_certificates) is C(true), then either I(platform_key), I(KEK),  I(database), or I(disallow_database) is required.

Reply:
Updated